recipes/libs/libxml2.yaml

```diff
@@ -0,0 +1,37 @@
+inherit: [meson]
+
+metaEnvironment:
+   PKG_DESCRIPTION: "XML toolkit from the GNOME project"
+   PKG_VERSION: "2.13.3"
+   PKG_LICENSE: "MIT"
+
+depends:
+   - libs::zlib-dev
+   - use: []
+     depends:
+      - libs::zlib-tgt
+
+checkoutSCM:
+   scm: git
+   url: https://gitlab.gnome.org/GNOME/libxml2.git
+   tag: "v${PKG_VERSION}"
+   commit: 3b1742b8391e966be780bdc43fdf959f7b3a118c
+
+buildTools: [target-toolchain, host-toolchain]
+buildScript: |
+   mkdir -p __src__/
+   rsync -a --delete $1/ __src__/
+   mesonBuild $(pwd)/__src__ \
+      -Dpython=false
+
+multiPackage:
+   dev:
+      provideDeps: ['*-dev']
+      packageScript: mesonPackageDev
+   tgt:
+      provideDeps: ['*-tgt']
+      packageScript: mesonPackageTgt $1 "!usr/share/gtk-doc" "!*.m4"
```

Comment:
Excluding `usr/share/gtk-doc` is not required any more since basement commit 32e9ecf5b4205f80dd3043424e57aedc34215054...